hello-oboe/app/src/main/java/com/google/example/hellooboe/MainActivity.kt

```diff
@@ -21,6 +21,8 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.MotionEvent
 import android.view.View
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
Don't use the synthetics, or at least don't import * with them. :)

Reply:
done, thanks!